### DISCUSSION TITLE
question: downshift combobox overlay or popover?

### DISCUSSION THREAD

**Discussion by eric-hc** [koolamusic/chakra-ui-autocomplete#94] [Q&A]
Would it be possible have the dropdown box render on top of any other elements? So that it doesn't push things down when dropdown is visible?

**Comment by koolamusic** [comment]
Yeah, I believe this can be achieved with a `zIndex` on any of the style props or the parent element

**Comment by eric-hc** [comment]
Tried doing both of the following, but no luck...does that look right?

```js
listStyleProps={{zIndex: '10'}}
inputStyleProps={{zIndex: '10'}}
```

**Comment by koolamusic** [comment]
yeah it does look right, 
Have you tried wrapping the dropdown in a `Box` and setting the box position to `relative` ? 
You can also attempt setting the `zIndex` on the Box something lik

```tsx
<Box zIndex={10} position="relative">
<CUIAutocomplete ... />
</Box>


```
